classes/class_dictionary.rst

```diff
@@ -60,6 +60,7 @@ To add a key to an existing dictionary, access it like an existing key and assig
 
 ::
 
+    d.waiting = 14 # Add String "waiting" as a key and assign the value 14 to it.
```

Comment:
BTW there should be two spaces before the `#` to match the style used in that code block.